examples/flows/hello-world-yaml/input-schema.yaml

```diff
@@ -0,0 +1,14 @@
+additionalProperties: false
```

Comment:
It's nice that we have a full example for a Flow's components defined in YAML. Once this is advertised better, I'm curious if folks stick with JSON or move to YAML